function/pkg/operator/generic_operator.go

```diff
@@ -9,21 +9,22 @@ import (
 )
 
 var (
+	//FIXME rename to GVR...
```

Comment:
I think '//TODO rename to GVR...' is better because IDEs find word 'TODO' and inform dev about it.

Reply:
I think it should be also visible with `//FIXME` (at least for vscode)
![image](https://user-images.githubusercontent.com/45931826/111633651-075a6000-87f6-11eb-84c9-c64ac23da2cc.png)